src/csv/area_and_circum_calculations.py

```diff
@@ -0,0 +1,27 @@
+import math
+
+
+def area_of_circles(radius):
```

Comment:
Function name here should not have been plural.  The behaviour of the function is to calculate area a circle given the radius.

Reply:
fixed